grafana/mixins.libsonnet

```diff
@@ -0,0 +1,3 @@
+{
+    mixins+:: {},
+}
```

Comment:
I'd move this into `config.libsonnet`, bit silly to have it as a separate file.

Reply:
Better actually to remove it. Currently, this lib is mixin un-aware. It can be used to implement mixins (e.g. as is done in `prometheus-ksonnet`, but they are not at this point relevant here.